tests/syncreload/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Handle command-line arguments
+source $KDBTESTS/flagparse.sh
+
+# Path to test directory
+testpath=/home/creid/devTorQ/bin/syncreload
```

Comment:
hardcoded file paths are a no-no - as mentioned above, these need to run without modification for future devs (otherwise, chances are future devs won't bother to run them and the unit tests become useless)